Users/Tutorial.rst

```diff
@@ -264,24 +267,34 @@ That looks horrible, doesn't it? Let's fix it!
     $ coala -S python.bears=PEP8Bear python.files=\*\*/\*.py \
     --apply-patches --save
     # other output ...
+    Executing section cli...
```

Comment:
good observation 👍 although this should be removed

Reply:
Since, we have just previously talked about the `cli` section, don't you think this will help the readers to follow the conversation in the tutorial?

Reply:
this should be removed in the sense that an issue must be created. no need for that when there is nothing in cli. It is good for now